cmd/xl-v1-utils.go

```diff
@@ -68,6 +68,9 @@ func reduceQuorumErrs(ctx context.Context, errs []error, ignoredErrs []error, qu
 	if maxCount >= quorum {
 		return maxErr
 	}
+	if maxErr == errDiskNotFound {
+		return SlowDown{}
```

Comment:
This shouldn't be converted here @poornas we should convert at toObjectErr()

Reply:
the reason for doing it here is that quorumErr is used as a default and errDiskNotFound will not bubble up to toObjectErr()

Reply:
It looks a bit odd as it may introduce bad practice, I would suggest we do more specific stuff here such as much like errDiskNotFound we should have errOperationTimedout such that top level can have better context.

Reply:
fixed this.